bluemira/equilibria/optimisation/problem/_nested_position.py

```diff
@@ -98,10 +96,7 @@ def __init__(
         targets: MagneticConstraintSet,
         position_mapper: PositionMapper,
         opt_algorithm="SBPLX",
-        opt_conditions={
-            "stop_val": 1.0,
-            "max_eval": 100,
-        },
+        opt_conditions: Optional[Dict[str, float]] = None,
```

Comment:
I'm not sure if it's better to do it like this, it's more clear potentially to see the opt_conditions as they were. Not sure how nb that is though.

Reply:
Its the mutable defaults again not for any readability improvements. I could possible put a debug statement with the contents of the opt_conditions like on L140 of wall silhouette
` ``python
       bluemira_debug(
            f"Setting up design problem with:\n"
            f"algorithm_name: {self.algorithm_name}\n"
            f"n_variables: {parameterisation.variables.n_free_variables}\n"
            f"opt_conditions: {self.opt_conditions}\n"
            f"opt_parameters: {self.opt_parameters}"
        )
```

Reply:
https://beta.ruff.rs/docs/rules/mutable-argument-default/

Reply:
I see yea.. Adding the defaults to the doc string would be fine as well

Reply:
sure I'll do that

Reply:
Ive done one better see updates